From 88f74ccb09824ecfc6666150bfeab13fd99a21c7 Mon Sep 17 00:00:00 2001
From: Jake Petroules <jake.petroules@qt.io>
Date: Mon, 12 Feb 2018 13:17:24 -0800
Subject: [PATCH] Add missing includes

Messages are sent to NSView and UIView pointers in this file, which
will generate unrecognized selector warnings (and eventually, errors).

Change-Id: I4c4d65b555eb4cac8d73596ccb986b14d34ddf31
Reviewed-by: Gabriel de Dietrich <gabriel.dedietrich@qt.io>
---
 .../avfoundation/mediaplayer/avfvideowindowcontrol.mm     | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/plugins/avfoundation/mediaplayer/avfvideowindowcontrol.mm b/src/plugins/avfoundation/mediaplayer/avfvideowindowcontrol.mm
index 4952551dc..5727cb0f4 100644
--- src/plugins/avfoundation/mediaplayer/avfvideowindowcontrol.mm.orig
+++ src/plugins/avfoundation/mediaplayer/avfvideowindowcontrol.mm
@@ -35,6 +35,12 @@
 
 #include <AVFoundation/AVFoundation.h>
 
+#if defined(Q_OS_MACOS)
+#import <AppKit/AppKit.h>
+#else
+#import <UIKit/UIKit.h>
+#endif
+
 QT_USE_NAMESPACE
 
 AVFVideoWindowControl::AVFVideoWindowControl(QObject *parent)
-- 
GitLab

